.github/workflows/pr-realm-js.yml

```diff
@@ -108,6 +108,17 @@ jobs:
         if: ${{ runner.os == 'Windows' }}
         uses: ilammy/msvc-dev-cmd@v1
 
+      - name: Setup CMake
+        if: ${{ runner.os == 'Windows' }}
+        uses: jwlawson/actions-setup-cmake@v2
+        with:
+          cmake-version: '3.29.2'
+
+      - name Check CMake
+        shell: bash
+        run: |
+          dir $(which cmake) >> $GITHUB_PATH
```

Comment:
This seems redundant? If `which` can find `cmake` why would it need to be added to the `PATH`?

From https://github.com/jwlawson/actions-setup-cmake/blob/master/README.md
> This action will update the path for your workflow to include cmake matching the platform and version requirements.

Reply:
True, but `cmake-js` fails when spawning `cmake` with `EINVAL`